docker-rpmbuild.sh

```diff
@@ -1,19 +1,28 @@
 #!/bin/bash
+set -e
+cd $(dirname $0)/install
```

Comment:
I was getting an error here that the install directory does not exist, and in this version it does not exist in the root of the repository. There is an install directory in the `slurm` dir, but since we cd into it, the rest of the script does not work (there is a `pwd` afterwards). So to get it to work i had to comment out line 2 to 6. Either there is a bug or i am not sure how to invoke the script. I ran it like : `docker-rpmbuild.sh` from the root of the repo. I tried running it from `slurm` directory as well, but as mentioned above, the `pwd` later on in line 18 fails.

Reply:
Agreed and fixed with latest push